pkgs/applications/video/manim-community/default.nix

```diff
@@ -0,0 +1,141 @@
+{ lib
+, fetchPypi
+
+, buildPythonApplication
+, pythonOlder
+
+, cairo
+, ffmpeg_4
+, sox
```

Comment:
if you are packing `v0.6.0`+, sox isn't required. https://docs.manim.community/en/stable/changelog/0.6.0-changelog.html#breaking-changes

Reply:
ok, done!